partner_helper/partner.py

```diff
@@ -39,19 +23,18 @@ def split_char(char, output_number, size):
     return result
 
 
-class ResPartner(orm.Model):
+class ResPartner(models.Model):
     _inherit = "res.partner"
 
     def _get_split_address(
-            self, cr, uid, partner, output_number, max_size, context=None):
+            self, partner, output_number, max_size):
         """ This method allows to get a number of street fields according to
             your choice. Default is 2 large fields in Odoo (128 chars).
             In some countries you may use 3 or 4 shorter street fields.
 
             example:
-            res = self.pool['res.partner']._get_split_address(
-                cr, uid, picking.partner_id, 3, 35, context=context)
-            address['street'], address['street2'], address['street3'] = res
```

Comment:
A) The previous comment shows how you may use the result of the function. I think it's good to let it.

B) It's always possible to improve: defining new variables seams more explicit than using an undefined list.

Do at least A).